app/dashboard/models.py

```diff
@@ -1692,12 +1711,13 @@ def view_props(self):
             obj = self.metadata['new_bounty']
         activity['title'] = obj.get('title', '')
         if 'id' in obj:
-            activity['bounty_url'] = Bounty.objects.get(pk=obj['id']).get_relative_url()
-            if activity.get('title'):
-                activity['urled_title'] = f'<a href="{activity["bounty_url"]}">{activity["title"]}</a>'
-            else:
-                activity['urled_title'] = activity.get('title')
-        activity['humanized_activity_type'] = self.humanized_activity_type
+            if 'category' not in obj or obj['category'] == 'bounty': # backwards-compatible for category-lacking metadata
```

Comment:
Hi @danlipert. I don't think so, since the body of this if statement applies only to activities of type bounty, as opposed to activities of type grant, whereas line 1720 should apply for both kinds of activities.

Reply:
@danlipert what's your opinion on this?

Reply:
@e18r Makes sense, my bad! Thanks!